core/src/main/scala/sttp/openai/OpenAi.scala

```diff
@@ -475,6 +485,7 @@ private object OpenAIEndpoints {
   val VariationsImageEndpoint: Uri = ImageEndpointBase.addPath("variations")
 
   def deleteFileEndpoint(fileId: String): Uri = FilesEndpoint.addPath(fileId)
+  def retrieveFileContentEndpoint(fileId: String): Uri = FilesEndpoint.addPath(fileId, "content")
```

Comment:
**suggestion:** Similar comment to what I've said on the delete method. Only the `.get()` call above determines that this is indeed a "retrieve" operation, the same endpoint may be used with over HTTP verbs for other purposes. Let's call this method just `fileContentEndpoint`.

Reply:
ah, now I get it. I think the previous comment that this should be called an `...Uri` not an `...Endpoint` applies